src/components/Prompt/Prompt.tsx

```diff
@@ -45,14 +51,22 @@ export const Prompt: FC<Props> = ({ when: shouldPrompt }) => {
       getWrapperAsWebComponent()?.addEventListener(triggerModalEvent, () => setIsCloseRecordModalOpen(true));
   }, []);
 
-  const blocker = useBlocker(({ nextLocation: { pathname, search } }) => {
+  const blocker = useBlocker(({ currentLocation, nextLocation: { pathname, search } }) => {
+    // TODO: investigate what's the case for this behavior
```

Comment:
Are we going to do it in scope of this ticket?

Reply:
I guess not